packages/node_modules/@ciscospark/react-redux-spark/src/spark.js

```diff
@@ -71,8 +71,13 @@ function defaultConfig(options = {}) {
     },
     credentials: {
       client_id: process.env.CISCOSPARK_CLIENT_ID,
+      federation: true,
```

Comment:
Are we expected to have federation enabled for all users of the widgets?

Reply:
![](https://media3.giphy.com/media/12eLSv8B5eN5sI/giphy.gif?cid=e1bb72ff5c7ee9507a5772794d5438f0)